.gitlab-ci.yml

```diff
@@ -740,8 +745,18 @@ test_backend_integration:
   after_script:
     - ${CI_PROJECT_DIR}/scripts/maybe-wait-in-stage.sh WAIT_IN_STAGE_TEST ${CI_PROJECT_DIR}/WAIT_IN_STAGE_TEST
     - export WORKSPACE=$(realpath ${CI_PROJECT_DIR}/..)
+    - ls ${CI_PROJECT_DIR}/../integration/backend-tests/results_*xml | xargs -n 1 -i cp {} .
+    - ls ${CI_PROJECT_DIR}/../integration/backend-tests/report_*html | xargs -n 1 -i cp {} .
```

Comment:
The `${CI_PROJECT_DIR}/../integration/backend-tests/results_*xml` wildcard will return that literal string if it fails to match, causing `cp` to try to copy that, and failing the whole command. Is that what is desired, or is having no reports ok? I guess not, but wanted to check. Using `nullglob` is an option, or maybe `find`.

Reply:
I was thinking about it as well.

The rest of the jobs do a plain `cp` as well, which means that they will fail if no reports are generated. So in that sense, this job is not worst than the others.

So I don't really know what should we do for these cases (failing on after_script). As it is now, the only symptom will be a status in the PR pending forever.

Reply:
Ok, it's good enough for now, I think.